### PR TITLE
generate test data for ontology operations

### PR DESCRIPTION
**Pull request by SepidehAlassi**
closes [DSP-229](https://dasch.myjetbrains.com/youtrack/agiles/110-4/111-29?issue=DSP-229)

### CODE REVIEW

**Comment by SepidehAlassi** [comment]
@nemmK can you please review this PR, whenever you have time?

File: webapi/src/main/scala/org/knora/webapi/SharedTestDataADM.scala [hunk 1]
```diff
@@ -1870,6 +1872,412 @@ object SharedTestDataADM {
             |  }
             |}""".stripMargin
     }
+    def createOntology(projectIri: IRI, label: String): String = {
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/SharedTestDataADM.scala:1875]: Style: Please put a blank line between two methods (here and elsewhere in the file).

File: webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala [hunk 2]
```diff
@@ -178,6 +180,20 @@ class OntologiesRouteV2(routeData: KnoraRouteData) extends KnoraRoute(routeData)
...
+      FastFuture.successful(
+          TestDataFileContent(
+            filePath = TestDataFilePath.makeJsonPath("update-ontology-metadata-request"),
+            text = SharedTestDataADM.changeOntologyMetadata(
+              ontologyIri, newLabel, newModificationDate
+            )
+        )
+      )
+    }
     private def getOntologyMetadataForProjects: Route = path(OntologiesBasePath / "metadata" / Segments) { projectIris: List[IRI] =>
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala:197]: add blank lines before method definitions

File: webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala [hunk 3]
```diff
@@ -329,7 +387,24 @@ class OntologiesRouteV2(routeData: KnoraRouteData) extends KnoraRoute(routeData)
             }
         }
     }
-
+    private def updateClassTestRequest: Future[Set[TestDataFileContent]] = {
+        var anythingLastModDate: Instant = Instant.parse("2017-12-19T15:23:42.166Z")
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala:391]: Change `var` to `val`

File: webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala [hunk 4]
```diff
@@ -296,6 +337,23 @@ class OntologiesRouteV2(routeData: KnoraRouteData) extends KnoraRoute(routeData)
         }
     }
 
+    private def createClassTestRequest: Future[Set[TestDataFileContent]] = {
+        var anythingLastModDate: Instant = Instant.parse("2017-12-19T15:23:42.166Z")
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala:341]: Change `var` to `val`

File: webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala [hunk 5]
```diff
@@ -564,6 +703,22 @@ class OntologiesRouteV2(routeData: KnoraRouteData) extends KnoraRoute(routeData)
             }
         }
     }
+    private def updatePropertyTestRequest: Future[Set[TestDataFileContent]] = {
+        var anythingLastModDate: Instant = Instant.parse("2017-12-19T15:23:42.166Z")
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala:707]: Change `var` to `val`

File: webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala [hunk 6]
```diff
@@ -363,7 +438,16 @@ class OntologiesRouteV2(routeData: KnoraRouteData) extends KnoraRoute(routeData)
             }
         }
     }
+    private def addCardinalitiesTestRequest: Future[TestDataFileContent] = {
+        var anythingLastModDate: Instant = Instant.parse("2017-12-19T15:23:42.166Z")
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala:442]: Change `var` to `val`

File: webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala [hunk 7]
```diff
@@ -398,6 +482,26 @@ class OntologiesRouteV2(routeData: KnoraRouteData) extends KnoraRoute(routeData)
         }
     }
 
+    private def replaceCardinalitiesTestRequest: Future[Set[TestDataFileContent]] = {
+        var anythingLastModDate: Instant = Instant.parse("2017-12-19T15:23:42.166Z")
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/routing/v2/OntologiesRouteV2.scala:486]: Change `var` to `val`

**Comment by benjamingeer** [comment]
The difference between `val` and `var`:

https://docs.scala-lang.org/overviews/scala-book/two-types-variables.html#the-difference-between-val-and-var

**Comment by SepidehAlassi** [comment]
> The difference between `val` and `var`:
> 
> https://docs.scala-lang.org/overviews/scala-book/two-types-variables.html#the-difference-between-val-and-var

thanks for the review, I just applied your suggestions. Can I merge now?

**Comment by nemmK** [comment]
I just finished the review and for me it looks fine with the requested change of the variable. The only thing I saw but you already mentioned it, is that the JS Lib Test is failing and I am not really sure why..

**Pull request merged**